packages/integration-karma/scripts/karma-plugins/env.js

```diff
@@ -30,7 +30,8 @@ function createEnvFile() {
         `    env: {`,
         `        NODE_ENV: "development",`,
         `        COMPAT: ${COMPAT},`,
-        `        NATIVE_SHADOW: ${NATIVE_SHADOW}`,
+        `        DISABLE_SYNTHETIC: ${DISABLE_SYNTHETIC},`,
+        `        NATIVE_SHADOW_ROOT_DEFINED: ${typeof ShadowRoot !== undefined}`,
```

Comment:
```suggestion
        `        NATIVE_SHADOW_ROOT_DEFINED: ${typeof ShadowRoot !== "undefined"}`,
```

Reply:
Thanks for catching that!

Reply:
Weird, I thought I committed this suggestion. Trying again.